_articles/send/create-send.md

```diff
@@ -0,0 +1,227 @@
+
+{% endcapture %}
+{{ web_vault | markdownify}}
+  </div>
+  <div class="tab-pane" id="browserextension" role="tabpanel" aria-labelledby="betab">
+{% capture browser_extension %}
+### Send from a Browser Extension
+
+Complete the following steps to create a new Send from a Bitwarden Browser Extension:
+
+1. Select the {% icon fa-paper-plane %} **Send** tab located at the bottom of your Browser Extension.
+
+   This view will list all Sends that haven't reached their [deletion date]({{site.baseurl}}/article/send-lifespan/#deletion-date). Like the {% icon fa-lock %} **My Vault** view, you can filter your Sends by selecting one of the available **Types**.
+2. To start creating a new Send, select the {% icon fa-plus %} **Add** icon:
+
+   {% image /send/send-browserext.png Send View in a Browser Extension %}
+
+3. On the **Add Send** view, specify the following to create the contents of your Send:
+
+   - **Name**: Specify an identifiable, meaningful name for this Send.
+   - **Type**: Specify whether this Send will be **Text** or a **File**:
+
+     |**Text** | Type or paste the desired text into the input box. Toggle the **When accessing the Send, hide the text by default** option to require recipients to [toggle visibility]({{site.baseurl}}/article/send-privacy/#hide-text) when they open a Send.|
+     |**File** | Select the **Choose File** button and browse for the file to send. The maximum file size per Send is 100 MB. (**Requires Premium**)|
+
+    By default, a Send will be scheduled for deletion 7 days from its creation. You may configure this and other options using the {% icon fa-chevron-down %} **Options** menu (see Step 4), otherwise select **Save** to finish creating your Send.
+
+4. Select the {% icon fa-chevron-down %} **Options** menu to configure the following options as desired:
+
+   |**Deletion Date**|The Send will be permanently [deleted]({{site.baseurl}}/article/send-lifespan/#deletion-behavior) on the specified date and time. By default, 7 days from creation. The **maximum allowed value** is 30 days from creation.|
+   |**Expiration Date**|The Send will [expire]({{site.baseurl}}/article/send-lifespan/#expiration-behavior) on the specified date and time. By default, **Never**.|
+   |**Maximum Access Count**|The Send will be [disabled]({{site.baseurl}}/article/send-lifespan/#maximum-access-count-behavior) after the specified access count is reached. By default, unspecified.|
+   |**Password**|[Require a password]({{site.baseurl}}/article/send-privacy/#passwords) to be entered by recipients of this Send in order to gain access.|
+   |**Notes**|Enter private notes for this Send, which will only be visible to the Sender.|
+   |**Disable this send so that no one can access it**|Check this box to prevent this Send from being accessibly to any recipients. You will still be able to interact this Send from your Send view.|
```

Comment:
@fschillingeriv, looks like these may just need to get updated in each client block on this page.

> **Disable this send so that no one can access it**

**Disable this Send so that no one can access it**

> Check this box to prevent this Send from being accessibly to any recipients. You will still be able to interact this Send from your Send view.

Check this box to prevent this Send from being accessible to any recipients. You will still be able to interact with this Send from your Send view.